js/form.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file
+ * Apple News individual entity vertical tab helper.
+ */
+
+(function ($, window, Drupal) {
+
+  Drupal.behaviors.AppleNewsEntityForm = {
+    attach: function attach() {
+      if (typeof $.fn.drupalSetSummary === 'undefined') {
+        return;
+      }
+
+      // Channels and sections checkboxes.
+      $('input[data-channel-id]').each(function() {
+        var _channel_id = $(this).data('channel-id');
+        if (!$(this).attr('checked')) {
+          // $('input[data-section-of="' + _channel_id + '"]').parent().hide();
```

Comment:
Was this copied from another place? If not, shall we remove it if it is not needed?

Reply:
yeah, I took from D7.  I will clean this one up.